Cargo.toml

```diff
@@ -15,7 +15,7 @@ geo-index = { version = "0.1", optional = true }
 geo-types = { version = "0.7" }
 num_enum = "0.7"
 num-traits = "0.2"
```

Comment:
Since we're not using `num_traits::FromPrimitive` anymore, I guess this dependency can be removed?
```suggestion
```